packages/whosmysanta-backend/src/data/index.js

```diff
@@ -1,17 +1,14 @@
 import mongoose from 'mongoose';
-import {mongoHost, mongoDatabase, mongoUser, mongoPassword} from '../constants';
 
-const host = mongoHost;
-const database = encodeURIComponent(mongoDatabase);
-const user = encodeURIComponent(mongoUser);
-const password = encodeURIComponent(mongoPassword);
+const host = process.env.MONGO_HOST;
+const database = encodeURIComponent(process.env.MONGO_DATABASE);
+const user = encodeURIComponent(process.env.MONGO_USER);
+const password = encodeURIComponent(process.env.MONGO_PASS);
 
-export const connectDatabase = () => {
+export default function connectDatabase () {
```

Comment:
Arrow functions have their purpose (lexical `this`) but `function` has its benefits too. For example, in this case, the `function` keyword allows us to write more concise, declarative code that also has the benefit of fitting on one line.